stories/topics/proc-aap-gcp-application-upgrade.adoc

```diff
@@ -0,0 +1,134 @@
+[id="proc-aap-gcp-application-upgrade"]
+
+Technical Preview
+~~~~~~~~~~~~~~~~~
+
+You can upgrade your existing {AAPonGCP} deployment to the newer version. The upgrade process covers upgrading of automation hub and automation controller. The upgrade process takes roughly the same amount of time as a {AAPonGCP} deployment install. Follow the steps below to ensure a smooth upgrade process.
+
+[NOTE]
+=====
+{AAPonGCP} supports sequential upgrades. All upgrades should be no more than one major version behind what you are currently upgrading to. For example, in order to upgrade to {AAPonGCP} to 2.2.20230215, you must be on the first version of {AAPonGCP}.
+=====
+
+.Prerequisites
+* You require the docker installed in order to run the upgrade playbook.
+
+== Preparing the environment
+
+The upgrade process requires a few volumes to be mounted, so *prepare a fresh directory* to be used for this process.
+
+.Procedure
+. First, create the directories that will be used to configure the upgrade process.
++
+[source,bash]
+----
+# Secrets directory. Store the GCP credentials file.
+$ mkdir secrets
+# Extra vars directory. All the configuration variables for the upgrade process.
+$ mkdir extra_vars
+----
++
+[NOTE]
+=====
++
+To create the GCP credentials file, see
+https://developers.google.com/workspace/guides/create-credentials  
++
```

Comment:
@leena-jawale its not formatted well
![image](https://user-images.githubusercontent.com/46899831/224367885-57497db5-c94d-4668-a81c-6da3f9362e1a.png)